common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -527,12 +541,14 @@ class @Problem
         answered = false
       if bind
         $(select_field).on 'change', (e) =>
-          @submitAnswersAndSubmitButton()
+          @submitAnswersAndSubmitButton false, true
           return
         return
 
     if answered
       @enableSubmitButton true
+      if data_changed
+        @saveNotification.remove()
```

Comment:
I looked through the usages, and there is only one where data_changed is false. And in that case, I don't see any harm in still calling @saveNotification.remove.